infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -2,24 +2,24 @@
 x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.46.0"
+  dpl-cms-release: "2024.47.1"
 x-webmaster: &webmaster-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
   dpl-cms-release: "2024.43.1"
-  moduletest-dpl-cms-release: "2024.46.0"
+  moduletest-dpl-cms-release: "2024.47.1"
 # Some sites on the webmaster plan wish to track the same release as
 # sites do per default in dpl-cms-release.
 x-webmaster-with-defaults: &webmaster-with-defaults-release-image-source
   # Reference default before webmaster anchor as YAML references will not
   # override existing keys.
   <<: *default-release-image-source
-  moduletest-dpl-cms-release: "2024.46.0"
+  moduletest-dpl-cms-release: "2024.47.1"
 x-webmasters-skipping-update-44.0: &webmaster-skipping-44
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.39.1"
-  moduletest-dpl-cms-release: "2024.46.0"
+  dpl-cms-release: "2024.46.0"
+  moduletest-dpl-cms-release: "2024.47.1"
```

Comment:
The naming here gets misleading but I cannot offer a perfect alternate solution.

If you agree that we need to address the naming could name it `x-webmasters-on-46.0` (which would be in line with the current naming) or `x-webmaster-herning` (which would align with the communication provided by the business).

Reply:
I kept the name as is because come next week I want to reconcile the diverged webmasters back in to one again. The current naming lets me keep track of that.